testplan/web_ui/testing/src/Report/reportUtils.js

```diff
@@ -29,7 +29,7 @@ function _mergeTags(tagsA, tagsB) {
       const tags = tagsB[tagName];
       if (tagsA.hasOwnProperty(tagName)) {
         let tagsArray = tags.concat(tagsA[tagName]);
-        let tagsSet = new Set(tagsArray);
```

Comment:
Reports can no longer use non-plain javascript types like `Set`. See [why](https://redux.js.org/style-guide/style-guide#use-plain-javascript-objects-for-state).